testsuites/store/src/main/java/org/eclipse/rdf4j/sail/InferencingTest.java

```diff
@@ -49,7 +49,7 @@ public void runTest(Sail sailStack, String subdir, String testName, boolean isPo
 		final String inputData = TEST_DIR_PREFIX + "/" + name + "-in.nt";
 		final String outputData = TEST_DIR_PREFIX + "/" + name + "-out.nt";
 
-		Collection<? extends Statement> entailedStatements = null;
+		Collection<? extends Statement> entailedStatements = new HashSet<>();
```

Comment:
Not clear why this is necessary. It seems to indicate that this test isn't going to work for your store and you should instead be ignoring this test.